NFT-RFC-002-use-cases.md

```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+
+In an attempt to solve this issue, Ruari’s organization setups what they are calling the New Labs Initiative Impact Bond, and have been able to get the promises of two other local charity groups in Los Angeles, where they will be piloting the program, which collectively have pledged a total of $10 million to the outcome fund. The three charities have collectively set outcome criteria as a function of both number of students serviced by the newly upgraded labs and the location of the school services so that higher rates of returns will be paid if the schools serviced are in what the outcome funding group has defined as schools in areas of “highest need.” In addition to these specific outcome criteria, the outcome funding group has also set a series of meta goals for the project as a whole where additional funding will be released once various percentages of schools have been serviced in the area with a target of reaching 75% of the identified “highest need” schools in five years. 
+
+Once the outcome funding has been pledged, Ruari approaches several companies in the local Los Angeles region and gets buy-in from four which agree to start upgrading the schools per the bond specifications. These companies take on the upfront capital investment costs of the upgrades which, when completed at a particular location, are evaluated by either a representative of the specific school’s administration or the county school board. This representative presents the company with a signed verification that the school was updated to the specified standards. This signed verification is wrapped as part of the verified claim which Company A eventually bundles and submits as a claim group that confirms they have upgraded labs as confirmed by the site verification entity. 
+
+The verifiable claims submitted by Company A prove that they upgraded labs which will service 10,000 students and that this covers 35% of the students in East Los Angeles County. They receive the specified payout from the outcome fund based on this claim bundle for both the site specific and meta-level metrics they achieved. This project continues over the next 5 years with various companies joining and leaving the project. Five years later the project was able to get 80% of the labs upgraded in the set of schools initially scoped by the project and Ruari is now going out to try and raise another round of outcome funding with the intention of taking this widely successful program to more cities around the country. 
+
 
 #### User Needs
+User needs here are broken down for each of the three users presented in the above story: Ellenor, Vicki and Stephen. 
+
+__Outcome Funder__
+* Configuration of a clear set of criteria on which to base the outcome payments
+* Assurance that reported outcomes are properly validated by a clear and mutually agreed upon validation method
+* Mechanism to make the outcome payments as the various companies who are fronting the upfront costs reach desired outcome levels
```

Comment:
We should separate the role of investor vs implementor, as this reduces conflicts of interest. Impact bonds would typically have capital market investors. In this user story, the companies doing the lab refurbs would more likely borrow the capital from banks against performance-based contracts for future outcome payments (which usually include at least a coupon value premium and often also include an additional incentive payment for performance).

The way bonds are structured, the capital investor/s take the financial risks on their capital principal for whether the outcome payment is released or not. The implementor may have at-risk the performance bonus.